index.html

```diff
@@ -0,0 +1,198 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- <link rel="stylesheet" href="/style/desktopAbout.css"> -->
+  <link rel="stylesheet" href="/style/mobileIndex.css">
+  <link rel="stylesheet" href="/style/DesktopIndex.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous"
+    referrerpolicy="no-referrer" />
+  <title>Alumni Club</title>
+</head>
+
+<body>
+  <div class="top-nav">
+    <div class="top-nav-1">
+        <a href="https://web.facebook.com/chukwuma.mosanya"><i class="fa-solid fa-facebook facebook-icon"></i></a>
+        <a href="https://twitter.com/ChukwumaMosanya"><i class="fa-solid fa-twitter twitter-icon"></i></a>
```

Comment:
Kindly fix these icons, they are not being displayed in the Webpage 🤗

![image](https://user-images.githubusercontent.com/73714615/191940146-9b107871-3b19-4055-a6a7-b654cc22f5be.png)